packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -56,71 +87,166 @@ export const pageQuery = graphql`
+See [When to use a Card](#when-to-use-a-card) and [Examples](#examples) for common Card patterns and dos/dont’s. A Card does not 
+currently handle any interactive events such as hover, click, or focus, though children 
+composed inside of it may commonly have event handlers.
 
-Acknowledging the flexibility of Card component, there are several non-negotiable
-properties of a Card that differentiate it from a more basic page-layout element, such as Box:
+Acknowledging the flexibility of Card component, there are several non-negotiable properties 
+of a Card that differentiate it from a more basic page-layout element, such as [Box](/utilities/box):
 
 * A background color of $color-background-body.
 * A border width of $border-width-20.
 * A border radius of $border-radius-20.
-* A border color of $color-border.
+* A border color of $color-border-light.
 * A flexible width and height.
     * Height is determined by a Card’s child components
     * Width is determined by the parent component or container surrounding Card
 
 ### Card vs. Box
 
-A card is a box with default attributes defined above. The purpose of the card is to provide a consistent experience to use across the website the design team can
+At its core, Card is a Box with styled with the specific default attributes above and more explicit use cases 
+and that you can find in Examples. If you find yourself limited by the default styling and constraints of a 
+Card, you may consider using a Box instead, but first consider bringing the problem you are trying to solve 
+to Design System Office Hours to see if another component or pattern could fit your needs. 
 
 
 ### Examples
 
+#### Basic Card
+
+An example of a Card with default padding.
+
+
 <LivePreview scope={{Card, Text, CardFooter, Heading}} language="jsx">
-  {`<Card>
-  <Heading as="h2">With a heading</Heading>
-  <Text as="p">Body</Text>
-  <CardFooter>
-    <Text as="p">I&apos;m | The | Footer</Text>
-  </CardFooter>
-</Card>`}
+  {`    <Card>
+      <Heading as="h2">This is a card</Heading>
+      <Text as="p">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut 
+        labore et dolore magna aliqua.
+      </Text>    
+    </Card>`}
 </LivePreview>
 
-<LivePreview scope={{Card, Text, CardFooter, Heading, Button}} language="jsx">
-  {`<Card>
-  <Heading as="h2">This is a card with buttons!</Heading>
-  <Text as="p">
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam dui tellus,
-    tristique in tincidunt cursus, efficitur at felis. Phasellus imperdiet,
-    lorem et commodo egestas, arcu.
-  </Text>
-  <CardFooter>
-    <Button variant="destructive">Cancel</Button>
-    <Button variant="submit">Submit</Button>
-  </CardFooter>
+#### Card with Title, Body and Button
+
+One of the most common use cases for a Card is to relate a Title (Heading) , supporting body copy (Text), 
```

Comment:
Should this say "(Paragraph)" instead of "(Text)"? @seanmateer

Reply:
+1 to that